content/docs/command-reference/checkout.md

```diff
@@ -207,8 +204,30 @@ $ md5 model.pkl
 MD5 (model.pkl) = 98af33933679a75c2a51b953d3ab50aa
 ```
 
-What happened is that DVC went through the DVC-files and adjusted the current
-set of <abbr>output</abbr> files to match the `outs` in them.
+DVC went through `dvc.lock` and adjusted the current set of <abbr>outputs</abbr>
+to match the `outs` in it.
+
+## Example: Specific targets
```

Comment:
Checking out only specific files or directories

Reply:
Done in all 3 refs with this example title.